cube/CMakeLists.txt

```diff
@@ -167,12 +158,11 @@ if (COMPILE_CUBE_SHADERS)
 endif()
 
 if(WIN32)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES")
+    add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
+    # vkcube / vkcube make use of M_PI and various other math defines.
+    add_compile_definitions(_USE_MATH_DEFINES)
```

Comment:
I've removed this from various KhronosGroup repos. Only cube actually seems to make use of this. If you remove this the build fails.